modules/Vtiger/ListView.php

```diff
@@ -272,6 +272,12 @@
 }
 $smarty->assign('advancedSearchMapResult', $advancedSearchMapResult);
 
+// GV responsible for showing or hiding filter panel
+$Application_ListView_FilterPanel_Open = GlobalVariable::getVariable('Application_ListView_FilterPanel_Open', '1', $currentModule, '', $_REQUEST['action']);
```

Comment:
I think the logic is inverted here
the default value is "1! but in that case you set the smarty variable to "display none"  
Is that correct?

Reply:
I set it to display none if it is not "1" `!$Application_ListView_FilterPanel_Open`